test/integration/fixtures/esm/loader-with-module-not-found/test-that-imports-non-existing-module.fixture.mjs

```diff
@@ -0,0 +1 @@
+import 'non-existent-package';
```

Comment:
Is this file loaded anywhere? Your test just uses the ts file.

Reply:
It is NOT loaded. But it is the reason for the test: the test checks that if a loader is present and an import of a non-existant package happens, then Mocha shows the correct error.

Reply:
We have a `*.fixture.ts` file which is to be tested. The loader resolves it to `*.fixture.js` as per your comment. But at the end it's `*.fixture.mjs` which throws the error? From `*.ts` to `*.js` then `*.mjs`?
Either your comment is incorrect, or the `*.fixture.mjs` has no purpose or I don't understand this loader test.

Reply:
The comment is incorrect. The loader resolves it to `test-that-imports-non-existing-module.fixture.mjs`, which is what throws the error.